test/library/standard/Path/prithvi-unnati/commonPath/testCommonPathStringArguments.chpl

```diff
@@ -0,0 +1,26 @@
+/*
+Test cases to validate the commonPath(paths: string ...?n) in Path.chpl
+*/
+
+use Path;
+
+//paths=null
+writeln(commonPath(""));
```

Comment:
Similarly to the empty array, it would be nice to test `commonPath()` (note: no arguments provided)

Reply:
But as the function is defined to have ```paths: string``` as an argument, the compiler throws an error for ```commonPath()```

Reply:
Huh, I must not have understood varargs correctly, I thought there was a way for zero args to be specified.  Not a big deal if not, but you ought to be able to remove a condition in your code if so